src/Order.js

```diff
@@ -0,0 +1,15 @@
+import RestaurantButton from "./RestaurantButton";
+import { useState } from "react";
+function Orders({ orderType }) {
+  const [orders, setOrders] = useState(0);
+  function incressOrder() {
```

Comment:
Can you can change this function(incressOrder) name to be more descriptive something like AddOrders or something?

Reply:
It's capital O in my code !